Motor motor1
Motor motor2
Motor motor3
Motor motor4
Servo pan
Servo tilt

float32 duration
